presto-grpc-udf/src/main/proto/GrpcUdfInvoke.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.facebook.presto.grpc.udf";
+option java_outer_classname = "GrpcUdfService";
+
+package udf;
+
+service GrpcUdfInvoke {
+  rpc InvokeUdf (GrpcUdfRequest) returns (GrpcUdfResult);
+}
+
+message GrpcFunctionHandle {
+  string functionName = 1;
+  repeated string argumentTypes = 2;
+  string returnType = 3;
+  string version = 4;
+}
+
+message GrpcSerializedPage {
+  bytes page = 1;
+  sint64 positionCount = 2;
+  sint64 uncompressedSizeInBytes = 3;
+  bytes pageCodecMarkers = 4;
+  sint64 checksum = 5;
+}
+
+message GrpcUdfRequest {
+  string source = 1;
+  GrpcFunctionHandle grpcFunctionHandle = 2;
+  GrpcSerializedPage inputs = 3;
```

Comment:
It's probably better to have another wrapper on page data, so in the future if we want to support other format like Arrow, it would be easily extensible.